Makefile

```diff
@@ -52,7 +52,7 @@ test-dirty: generate
 	# TODO: also check that there are no untracked files, e.g. extra .go and .yaml ones
 
 .PHONY: test
-test: check-fmt lint vet test-unit test-e2e
+test: check-fmt lint vet test-unit test-e2e shellcheck
```

Comment:
[optional] I would suggest putting `shellcheck` between `vet` and `test-unit`, so that we don't have to wait for the very long `test-e2e` to finish to see any errors in shell scripts